### ISSUE TITLE
RSS Feed broken

### ISSUE THREAD

**Issue by schiessle** [nextcloud/nextcloud.com#85]
Hi,
I was told that the RSS feed (https://nextcloud.com/feed/ ) is broken. I just tested it and indeed, I get a "invalid XML" error when I try to add the feed to my feed reader.

cc @LukasReschke @jospoortvliet

**Comment by LukasReschke** [comment]
Did some changes that at least SimplePie accepts this now: http://simplepie.org/demo/?feed=https://nextcloud.com/feed/

Can you recheck?

**Comment by Bugsbane** [comment]
Still broken for me. I came here to report this. Also, [the feed for releases](https://nextcloud.com/category/release/feed/) listed on [the news page](https://nextcloud.com/news/), while seemingly being valid, doesn't list any releases :(. Should I file another issue for that, or could this be related?

**Comment by tobiasKaminsky** [comment]
At least the junk after the closing < /rss> needs to be removed.

**Comment by jospoortvliet** [comment]
I'm afraid to summon @LukasReschke because some day he's going to tell me where this code lives and how to fix it... and then I'm on the hook.

**Comment by LukasReschke** [comment]
> I'm afraid to summon @LukasReschke because some day he's going to tell me where this code lives and how to fix it... and then I'm on the hook.

And @LukasReschke is afraid to tell you that this will stay like that until we either have someone that has time for this or migrate to something debuggable instead of Wordpress 🚀

**Comment by jospoortvliet** [comment]
Right, also see https://github.com/nextcloud/nextcloud.com/issues/82#issuecomment-264669519
